_posts/2018-08-28-announcing-riff-0-1-2-on-Knative.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: single
+title: "Announcing riff v0.1.2 on Knative"
+header:
+  overlay_image: /images/liverpool-st.jpg
+excerpt:
+  With verbose builds and function chaining using Channels and Subscriptions    
+permalink: /blog/announcing-riff-0-1-2-on-Knative/
+---
+
+[riff v0.1.2](https://github.com/projectriff/riff/releases/tag/v0.1.2) on Knative is now available. Many thanks to all riff and Knative contributors.
+
+## install
```

Comment:
The previous blog post had "regular" Title casing for titles. This one uses all lowercase, is this intentional?

Reply:
yes - we are rebels

Reply:
I personally don't like it (although I like thinking I'm a rebel, sometimes). I think it hursts readability, because the visual cues of the structure of the doc are lost